.eslintrc.yml

```diff
@@ -65,3 +68,8 @@ overrides:
     rules:
       max-len: "off"
       func-names: "off"
+  - files:
+      - "test/*.ts"
+      - "*.js"
+    rules:
+      "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
Why not put this rule after l.65 for the files: 
```
      - "tests/*.ts"
      - "test/*.ts"
      - "*.spec.ts"
```

Also, not sure `tests/*.ts` and `*.js` are still necessary